nuget/AppCenterCrashes.nuspec

```diff
@@ -47,6 +47,7 @@
     <!-- targets files add correct references to proper platforms -->
     <file src="$uwp_dir$/Microsoft.AppCenter.Crashes.targets" target="build/uap10.0"/>
     <file src="$uwp_dir$/Microsoft.AppCenter.Crashes.dll" target="ref/uap10.0" />
+    <file src="$uwp_dir$/x86/Microsoft.AppCenter.Crashes.xml" target="ref/uap10.0" />
```

Comment:
The x86 is weird but it works e2e when project configured on x64, I just tested 😅

Reply:
Yeah, since we don't know the correct location of the xml file exist on CI, I asked Dmitry's help for compare the files of uap and x86, and confirmed they are same. So this can be used as a work around.